src/database/migrations/0042-create-base_fields_copy_tasks.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE base_fields_copy_tasks (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  status task_status NOT NULL,
```

Comment:
The [`capitalisation.types` rule](https://docs.sqlfluff.com/en/stable/reference/rules.html#rule-capitalisation.types), by default, wants us to be consistent within a file about using uppercase or lowercase type names. Unfortunately, it does not distinguish between PostgreSQL-defined and user-defined types, so it's not happy about a snippet like:

```sql
CREATE TABLE x (
  funder_short_code short_code_t NOT NULL REFERENCES funders(short_code) ON DELETE CASCADE,
  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
...
```

because `short_code_t` is all lowercase and `TIMESTAMP WITH TIME ZONE` is all caps.

I see a few possible solutions:
1. use uppercase everywhere, including user types: `SHORT_CODE_T`
2. use lowercase everywhere, including SQL types: `timestamp with time zone`
3. add our user-defined types to the ignore list for this rule in the SQLFluff configuration, and use uppercase everywhere else

I'm leaning towards option 2, lowercase everywhere, but want to check in - what do you all think about that?

Reply:
I like (2) as well.